src/Resources/contao/dca/tl_page.php

```diff
@@ -43,6 +43,7 @@
 	'exclude'		=> true,
 	'inputType'		=> 'checkbox',
 	'eval'			=> array('tl_class'=>'w50 m12'),
+    'sql' => "char(1) NOT NULL default ''"
```

Comment:
Since this is Contao 4.5+ only, we can use Doctrine array notation for SQL. I think that would be cool to lead by example. 

DBAL Docs: https://doctrine-dbal.readthedocs.io/en/latest/reference/schema-representation.html#column

Should be something like this:
`'sql' => ['type'=>'string', 'length' => 1, 'notnull' => true, 'fixed' => true, 'default' => '']`